syntax = "proto3";

message DeeprouteTrackingEvalConfig{
    Deeproute_Eval_Input_Reader deeproute_eval_input_reader = 1;
}


message Deeproute_Eval_Input_Reader{
    repeated string interest_labels = 1;
    repeated uint32 filter_points_number = 2;
    repeated uint32 distance_thresh = 3;
    repeated float overlap_thresh = 4;
    float occlusion_thresh = 5;
    uint32 num_features_for_pc = 6;
    uint32 distance_compute_model = 7;
    repeated uint32 velocity_thresh = 8;
    Sampler1 name_to_label = 9;
}

message Sampler1{
    repeated Group1 sample_groups = 1;
}

message Group1{
    map<string, uint32> name_to_num = 1;
}
